front/public/locales/fr/rollingstock.json

```diff
@@ -19,11 +20,17 @@
     "all": "Toutes les courbes",
     "default": "Courbe par défaut"
   },
+  "delete": {
+    "comfortLevels": "Êtes-vous sûr de vouloir supprimer les courbes correspondant à ce niveau de confort ? Tous les modes de tractions et profils électriques associés seront perdus.",
+    "electricalProfiles": "Êtes-vous sûr de vouloir supprimer les courbes correspondant à ce profil électrique ?",
+    "powerRestrictions": "Êtes-vous sûr de vouloir supprimer la courbe correspondant à ce code de restriction ?",
+    "tractionModes": "Êtes-vous sûr de vouloir supprimer les courbes correspondant à ce mode de traction ? Toutes les courbes associées à ce mode, y compris dans les autres niveaux de confort, seront perdues."
+  },
```

Comment:
You forgot to add this in the english file

```
  "delete": {
    "comfortLevels": "Are you sure you want to delete this comfort level ? All the curves associated to it will be lost.",
    "electricalProfiles": "Are you sure you want to delete this electrical profile ? All the curves associated to it will be lost.",
    "powerRestrictions": "Are you sure you want to delete this power restriction ? All the curves associated to it will be lost.",
    "tractionModes": "Are you sure you want to delete this traction mode ? All the curves associated to it will be lost."
  },
```